CMakeLists.txt

```diff
@@ -28,8 +28,8 @@
 # The SPIR-V headers from the SPIR-V Registry
 # https://www.khronos.org/registry/spir-v/
 #
-cmake_minimum_required(VERSION 3.0)
-project(SPIRV-Headers VERSION 1.5.5)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
[KhronosGroup/Vulkan-Headers](https://github.com/KhronosGroup/Vulkan-Headers) uses 3.14 so it should be safe for `spirv-headers`. Also `spirv-tools` uses 3.17 as it's minimum